client/src/App.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from "react";
 import axios from "axios";
 import { Route } from "react-router-dom";
+import "./App.css";
 // components
```

Comment:
uh...duh? the path has components in them especially in the App. this isnt needed imo